packages/components/src/components/ResizableBox/ResizableBox.tsx

```diff
@@ -52,67 +48,116 @@ const Resizers = styled.div<ResizersProps>(
         ${$maxHeight && `max-height: ${$maxHeight}px;`};
         box-sizing: border-box;
         position: relative;
+        ${
+            $isResizing &&
+            `user-select: none;
+            -webkit-user-select: none;`
+        };
+    `,
+);
 
-        ${$isResizing && `user-select: none;`};
+const handlersCommonStyles = () => css`
```

Comment:
I think this could be variable

Reply:
Not sure how I could use `$zIndex` in there then.